CMakeLists.txt

```diff
@@ -98,6 +99,7 @@ else()
   set(no_whole_archive_flag "--no-whole-archive")
 endif()
 
+set(Boost_USE_MULTITHREADED      ON)
```

Comment:
what's this for?

Reply:
If you have both single/multithread boost libraries installed, it would choose the multithread ones. I got a lot cmake warning messages without it.

Reply:
The default value of `Boost_USE_MULTITHREADED` is documented as ON.

Reply:
ah okay, `Boost_USE_MULTITHREADED` is default ON when using cmake's module, but
> The default is to use either.

for Boost's module.